src/rpc/evo.cpp

```diff
@@ -1430,13 +1445,28 @@ static UniValue protx_info(const JSONRPCRequest& request)
         g_txindex->BlockUntilSyncedToCurrentChain();
     }
 
+    CBlockIndex* pindex{nullptr};
+
     uint256 proTxHash(ParseHashV(request.params[0], "proTxHash"));
-    auto mnList = deterministicMNManager->GetListAtChainTip();
+
+    if (request.params[1].isNull()) {
+        LOCK(cs_main);
+        pindex = ::ChainActive().Tip();
```

Comment:
instead `::ChainActive()` use `chainman.ActiveChain().Tip()`  which is easily available in RPC code through:
```cpp
    const ChainstateManager& chainman = EnsureAnyChainman(request.context);
```